### DISCUSSION TITLE
AI，人类，与世界

### DISCUSSION THREAD

**Discussion by qingyuan114514** [viopsa233/AI-testers-Forum#18] [主意,想法]
【超长图警告】我与Claude就世界和宇宙的探讨
![QQ图片20230422222127](https://user-images.githubusercontent.com/104309488/233790457-aa5bad0d-0c56-49d9-9318-7d078b457e95.png)
贴友们没兴趣会删帖，不用低创占用资源

**Comment by viopsa233** [comment]
看懵了😵

**Comment by youtehub** [comment]
这个好像也是基于chatgpt 3.5开发的，测试过无法回答2021年之后的问题

**Reply by viopsa233** [reply]
可以手动喂资料给她，链接也可以

**Comment by 22aa44** [comment]
感觉可以的呀

**Comment by araea** [comment]
主观与客观是哲学上的两个重要概念。主观是指依赖于个人的感受，想法，判断或偏好的事物。客观是指不依赖于个人的感受，想法，判断或偏好的事物。主观与客观之间的关系是一个复杂而有争议的话题，不同的哲学流派有不同的观点和论证。
